src/languages/en.ts

```diff
@@ -754,6 +754,7 @@ export default {
             atLeastTwoDifferentWaypoints: 'Please enter at least two different addresses.',
             splitExpenseMultipleParticipantsErrorMessage: 'An expense cannot be split between a workspace and other members. Please update your selection.',
             invalidMerchant: 'Please enter a correct merchant.',
+            workspaceNotAccessible: 'The workspace is no longer accessible. Please try again on a different workspace.',
```

Comment:
@dominictb [This](https://github.com/Expensify/App/issues/42950#issuecomment-2158857654) is the suggestion for the error message.